custom-typings/debounce.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'debounce' {
+    const debounce: <A extends Function>(f: A, interval?: number, immediate?: boolean) => A & { clear(): void; };
+    export = debounce;
+}
```

Comment:
thats not good. 
https://www.npmjs.com/package/@types/debounce

Reply:
this package is not compatible with this module. I have tried